lib/main.dart

```diff
@@ -135,7 +135,7 @@ class HomePage extends StatelessWidget {
               SizedBox(height: 118, child: Center(child: HorProgressBar())),
               SizedBox(height: 118, child: Center(child: HorProgressBar())),
               SizedBox(height: 118, child: Center(child: HorProgressBar())),
-              SizedBox(height: 30, child: Center(child: addButton()))
+              //SizedBox(height: 30, child: Center(child: addButton()))
```

Comment:
Could cause a merge conflict here, But if it does just change 'addButton()' to 'AddButton()'